Makefile

```diff
@@ -77,7 +82,8 @@ docs-build:
 .PHONY: start
 # Run both the back-end and the front end
 start:
-	$(MAKE) -j 2 start-backend start-frontend
```

Comment:
I won't generalise this for training/educational purposes. So the simple approach remains the default, then have the "advanced" deployment testbed.

Reply:
I'm not quite following you here.  Can you clarify and LMK if there's I should do here?